src/main/java/fruit/shop/records/parser/RecordsSaverCommaSeparated.java

```diff
@@ -0,0 +1,34 @@
+package fruit.shop.records.parser;
+
+import fruit.shop.strategy.ActivityStrategy;
+import fruit.shop.strategy.ActivityStrategyImpl;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RecordsSaverCommaSeparated implements RecordsSaver {
+    public static final String COMMA = ",";

```

Comment:
don't name comma a COMMA, use the naming of the constant as the opportunity to improve readability for the user that will be reading this code the first time (for example describing its role - DELIMITER). Remembers, developers spend more time reading others' code than writing their own